### PR TITLE
{bio} [goolf-1.4.10] ms-20071014 (REVIEW)  

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/m/ms/ms-20071014-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+# as no version is provided in the tarball I verify checksum is fine
+checksums = [('md5', 'ef693dcc424fd5293b0bea4c68953700')]
+
+# delete provided binary
+postinstallcmds = ["rm -fr %(installdir)s/msdir/ms"]
+
+# here is where we do the compilation 
+# I got this line from the provided build script in %(installdir)s/msdir/clms
+postinstallcmds += ["cd %(installdir)s/msdir && $CC -O2 -o ms  ms.c  rand1.c streec.c -lm"]
```
Comment (boegel) [inline easybuild/easyconfigs/m/ms/ms-20071014-goolf-1.4.10.eb]: why not make this `CmdCp` instead?

**Comment by boegel** [comment]
@pescobar: please submit a test report too (but take a look at the remark first)

File: easybuild/easyconfigs/m/ms/ms-20071014-goolf-1.4.10.eb (hunk 1 above)
Comment (pescobar) [inline easybuild/easyconfigs/m/ms/ms-20071014-goolf-1.4.10.eb]: IMHO, this one is easier to read/understand than the CmdCp and anyway I was going to use files_to_copy = [ " * " ] so why not using TarBall easyblock? In any case, I think it's just a matter of taste, isn't it?

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f05fa9ce8e8d1fcd7c37 for a full test report.

File: easybuild/easyconfigs/m/ms/ms-20071014-goolf-1.4.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/m/ms/ms-20071014-goolf-1.4.10.eb]: you're duplicating lots of stuff here that's already done by `CmdCp`: cleanup, you can copy stuff to `bin`

you're right that `CmpCp` should be enhanced though, it's usage is now rather strange

Comment (fgeorgatos) [inline easybuild/easyconfigs/m/ms/ms-20071014-goolf-1.4.10.eb]: IMHO, `$CC` in `postinstallcmds` is no good should be avoided at all costs:

in the future, we might wish to transfer ownership of files to a separate user, after install step, as means of isolation.
I am already doing something of the like, in a manual step, in order to stabilise the builds and make the immutable.
Juelich is also doing something of the like, whereby one team passes buildset ownership to another.

In short, please do not change things after the install step, if you have a choice ;-)

Comment (boegel) [inline easybuild/easyconfigs/m/ms/ms-20071014-goolf-1.4.10.eb]: Good argumentation to go with `CmdCp`?
